support-frontend/app/controllers/Application.scala

```diff
@@ -114,7 +114,10 @@ class Application(
   private def contributionsHtml(countryCode: String, idUser: Option[IdUser])
     (implicit request: RequestHeader, settings: AllSettings) = {
 
-    val css = CSSElementForStage(assets.getFileContentsAsHtml, stage, RefPath("newContributionsLandingPageStyles.css"))
+    val elementForStage = CSSElementForStage(assets.getFileContentsAsHtml, stage)_
```

Comment:
if preferred, I could have kept this as 1 parameter group and done something more like this:
```suggestion
    val elementForStage = CSSElementForStage(assets.getFileContentsAsHtml, stage, _)
```